ROADMAP.md

```diff
@@ -1,10 +1,58 @@
 # Kubeflow Roadmap
 
 ## Kubeflow 1.5 Release, Due: February 2022
-* Kubeflow 1.5 timeline is tracked in [#538](https://github.com/kubeflow/community/pull/538)
-* Additional details will be posted in early 1Q'22
+* Kubeflow 1.5 [milestones and timeline](https://github.com/kubeflow/community/pull/538)
+
+### Themes
+* Increase ML pipeline functionality with aggregation and passing of parameters and artifacts between pipeline components
+* Improve model accuracy and reduce overfitting, especially with hyper parameter tuning
+* Simplify operations and optimize utilization (including spot instance use cases for distributed training) 
+* More consistent user experience - UI appearance, features and naming
+* Improved documentation, tutorials and examples
+* Stretch - Support for K8s 1.22 and associated dependencies (cert mgr, istio)
+
+### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)
+
+#### Kubeflow Pipelines, v1.8
+* [Emissary executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) is default rather than Docker runtime
+* Advanced features in [KFP v2 Control Flow](https://docs.google.com/document/d/1TZeZtxwPzAImIu8Jk_e-4otSx467Ckf0smNe7JbPReE/edit?resourcekey=0-lTeZGW_Ys78j1LU60CEARg#heading=h.x9snb54sjlu9)
+   * Aggregation of parameters and artifacts in DAG or sub-DAG
+   * Sub-DAG outputs (parameters and artifacts) can be inputs to the next sub-DAG
+   * Increase exit handler functionality with conditionals and sub-DAG options
```

Comment:
Hello Josh, the KFPv2 is not planned for this release, therefore please remove the content here. 

The tracking bug for KFP 1.8 is here https://github.com/kubeflow/pipelines/issues/6015